### ISSUE TITLE
Feature Request : Add Disabled State for Tabs

### ISSUE THREAD

**Issue by Sparsh-Malhotra** [rewindui/rewindui#70]
Hey, really nice library. I have a feature request that I believe will make it even better! Currently, the tabs component lacks a disabled state, which limits its accessibility and user experience in certain scenarios. When a tab is in the disabled state, it should be unclickable and should be skipped during keyboard navigation. I would love to take this issue and contribute to this amazing library.

**Comment by NickDunas** [comment]
Hello @Sparsh-Malhotra and thank you for your kind words!

I was almost certain that I did include a disabled state, but looks like you are right and I completely missed it.

Sure feel free to implement this and open a pull request! 😄
